src/main/java/qwerty/QwertyException.java

```diff
@@ -0,0 +1,11 @@
+package qwerty;
+
+/**
+ * This class encapsulates exceptions unique to the Qwerty chatbot.
+ * An example is missing or invalid arguments.
+ */
+public class QwertyException extends Exception {
```

Comment:
Perhaps this exception class could be named less general to reflect its use cases?